src/site/content/en/lighthouse-accessibility/offscreen-content-hidden/index.md

```diff
@@ -1,9 +1,11 @@
 ---
```

Comment:
Can't comment on it because it's not part of this PR but I noticed this doc opens with:

"For instance, content that's offscreen or just presentation should be hidden from a screen reader."

I think it should be: "For instance, content that's offscreen or just **presentational** should be hidden from a screen reader."